jekyll/_cci2/runner-config-reference.adoc

```diff
@@ -71,13 +71,34 @@ If specifying a custom script, it must:
 1. Take great care to ensure the task agent (that is, the arguments passed to it) runs
 2. Forward the exit code from task agent as its own exit code
 
-Example:
+Example with `sudo` prefix:
 
+- config snippet
 ```yaml
 runner:
   command_prefix: ["sudo", "-niHu", "USERNAME", "--"]
 ```
 
+Example with custom script prefix:
+
+- config snippet
+```yaml
+runner:
+  command_prefix: ["/opt/circleci/wrapper.sh"]
+```
+
+- `/opt/circleci/wrapper.sh`
```

Comment:
@lewang Just want to verify that this is meant to introduce the bash script below since it looks like it belongs to the code snippet above instead. Thanks! I'll approve and just wait for confirmation 👍